legal-api/src/legal_api/services/document_meta.py

```diff
@@ -230,6 +232,27 @@ def get_corrected_ia_reports(self, filing: dict):
 
         return reports
 
+    def get_transition_reports(self):  # pylint: disable=no-self-use
+        """Return transition meta object(s)."""
+        reports = []
+
+        if self.is_completed():
```

Comment:
I think this should show the transition application in the documents if the filing is in the paid status. Please check with Linda